services/notifications/pkg/config/config.go

```diff
@@ -28,6 +28,7 @@ type Notifications struct {
 	Events            Events                `yaml:"events"`
 	MachineAuthAPIKey string                `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY;NOTIFICATIONS_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary to access resources from other services."`
 	EmailTemplatePath string                `yaml:"email_template_path" env:"OCIS_EMAIL_TEMPLATE_PATH;NOTIFICATIONS_EMAIL_TEMPLATE_PATH" desc:"Path to Email notification templates overriding embedded ones."`
+	TranslationPath   string                `yaml:"translation_path" env:"NOTIFICATIONS_TRANSLATION_PATH" desc:"(optional) Set this to a path with custom translations to overwrite the builtin translations. See the documentation for more details."`
```

Comment:
Question to @mmattel and @micbar

Does it make sense to have an envvar `OCIS_TRANSLATION_PATH` where I can store all translations for all services? Or we want to keep seperated?

Reply:
:+1: for `OCIS_TRANSLATION_PATH`

Reply:
I also prefer a global envvar.
In such a case we MUST ensure that file and folder names are clear and do not conflict.

I therefore would propose the following:
* a common root path defined by `OCIS_TRANSLATION_PATH`
* subfolders making the usecase clear
BAD: `shares/shareCreated.email.body.tmpl`
GOOD: `email/shares/shareCreated.email.body.tmpl`

In the _good_ example, this would end up `OCIS_TRANSLATION_PATH/email/shares/shareCreated.email.body.tmpl`

We then also need to update eg the userlog service:
Current: `{USERLOG_TRANSLATION_PATH}/de_DE/LC_MESSAGES/userlog.po`
New: `{USERLOG_TRANSLATION_PATH}/<whatever_we_define_here>de_DE/LC_MESSAGES/userlog.po`

As you can see, the first subfolder in `OCIS_TRANSLATION_PATH` would define the usecase.

Reply:
You are mixing up `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` and `NOTIFICATIONS_TRANSLATION_PATH`

- `NOTIFICATIONS_TRANSLATION_PATH` is used for the translations. When using the global var all translation would live in the same folder. `{OCIS_TRANSLATION_PATH}/de_DE/LC_MESSAGES/userlog.po` and `{OCIS_TRANSLATION_PATH}/de_DE/LC_MESSAGES/notifications.po` would live in the same folder.
- `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` is used for the email template. This is unique to notifications service (the global var makes no sense). It the email template is used to send html emails (instead of plain text ones)